src/node_api.cc

```diff
@@ -25,6 +25,22 @@ napi_env JsEnvFromV8Isolate(v8::Isolate* isolate) {
   return reinterpret_cast<napi_env>(isolate);
 }
 
+// convert from n-api property attributes to v8::PropertyAttribute
+v8::PropertyAttribute V8PropertyAttributesFromAttributes(
```

Comment:
I’d suggest adding `static inline` since there’s no reason to expose this to the outside world